uint64 timestamp
float64 accel_x
float64 accel_y
float64 accel_z
